test/integration/price-import.spec.coffee

```diff
@@ -87,16 +87,24 @@ describe 'Price Importer integration tests', ->
     prices = [
       {
         sku: 'sku1'
-        value:
-          centAmount: 9999
-          currencyCode: 'EUR'
+        prices: [
+          {
+            value:
+              centAmount: 9999
+              currencyCode: 'EUR'
+          }
+        ]
```

Comment:
If it's like this then ok. I thought you mentioned that for `I151001_6`, the `I151001` identifies a product and `_X` the variant. So that's why I asked.